### PR TITLE
Ensure correct balance decrease during settlement

### PR DESCRIPTION
**Pull request by chompomonim**
Settlement without channel rebalancing should decrease balance for provider with partial stake.

### CODE REVIEW

File: contracts/HermesImplementation.sol [hunk 1]
```diff
@@ -203,13 +203,13 @@ contract HermesImplementation is FundsRecovery {
             _increaseStake(_channelId, _stakeIncrease, true);
             _amountToSettle = _amountToSettle.sub(_stakeIncrease);
         } else {
-            // Update balance and decrease hermes locked funds.
-            _channel.balance = _channel.balance.sub(_unpaidAmount);
+            // Dcrease hermes locked funds.
```
Comment (soffokl) [inline contracts/HermesImplementation.sol:206]:
```suggestion
            // Decrease hermes locked funds.
```

**Comment by chompomonim** [comment]
@soffokl @vkuznecovas any thoughts on why a build is failing with such a message?

```Service 'psc' failed to build: The command '/bin/sh -c npm run compile' returned a non-zero code: 1
{"level":"info","time":"2020-07-04T13:06:10+02:00","message":"Could not start containers"}
exec: docker-compose down
Removing network payments-smart-contracts_default
Error: running "docker-compose up --build -d" failed with exit code 1
exit status 1
Running after script...
$ sudo chown -R gitlab-runner:gitlab-runner $GOPATH
ERROR: Job failed: exit status 1```

**Comment by vkuznecovas** [comment]
@jaro Error: Cannot find module 'eth-json-rpc-filters'. I think you forgot to install it.

**Comment by vkuznecovas** [comment]
Let's merge this in anyway, i'll fix the build afterwards.

**Pull request merged**